lib/mapbox/geocode.py

```diff
@@ -194,15 +208,14 @@ def reverse_geocode(latitude, longitude):
 
         results = response_json.get('features', [])
         if len(results) > 0:
-            result = results[0]
```

Comment:
We go with `results[0]` because that is the most accurate/highest confidence result from the API.